pkg/karpenter/providers/helm.go

```diff
@@ -0,0 +1,27 @@
+package providers
+
+import (
+	"bytes"
+	"context"
+
+	"helm.sh/helm/v3/pkg/getter"
+)
+
+// Getter is an interface to support GET to the specified URL.
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 -generate
+//counterfeiter:generate -o fakes/fake_helm_getter.go . Getter
+type Getter interface {
```

Comment:
I thought this was a helm getter at first glance, worth renaming to `HTTPGetter`? Or `URLGetter`? Idk `Getter` is generic, but also my suggestions are crap :joy:

Reply:
Good call. I wanted to keep it the same way Helm names it so be less confusing. But managed to make it more confusing I guess. :)

Reply:
will fix in next pr